docs/ref/contrib/admin/index.txt

```diff
@@ -1326,15 +1326,19 @@ templates used by the :class:`ModelAdmin` views:
 
 .. warning::
 
-    :meth:`ModelAdmin.save_model` and :meth:`ModelAdmin.delete_model` must
-    save/delete the object, they are not for veto purposes, rather they allow
-    you to perform extra operations.
+    When overriding :meth:`ModelAdmin.save_model` and
+    :meth:`ModelAdmin.delete_model`, your code must save/delete the
+    object. They are not meant for veto purposes, rather they allow you to
+    perform extra operations.
 
 .. method:: ModelAdmin.save_model(request, obj, form, change)
 
     The ``save_model`` method is given the ``HttpRequest``, a model instance,
     a ``ModelForm`` instance and a boolean value based on whether it is adding
-    or changing the object. Here you can do any pre- or post-save operations.
+    or changing the object. By overriding this method, your ``ModelAdmin``
+    subclass can do pre- or post-save operations. Your code should call
+    ``super().save_model()`` to save the object. If that's not appropriate for
```

Comment:
Currently ModelAdmin's ``save_model()`` only calls ``obj.save()``. If that were to change, the extra behavior might not be desired in the subclass. I do not have a specific example, but if the docs suggest calling ``super()``, it still should be documented that ``obj.save()`` needs to be called if ``super()`` isn't.